Rakefile

```diff
@@ -1,6 +1,6 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to(ENV["YAST_SUBMIT"] ? ENV["YAST_SUBMIT"].to_sym : :sle12sp2)
```

Comment:
I'd still keep `YAST_SUBMIT` handling in place, it might be useful in some cases...